internal/handlers/common/tail.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Tail creates a tail for sql find and count queries.
```

Comment:
Somehow that became a package comment, not a func comment.

And I think defining a term (tail) via the same term is not helpful.

```suggestion
// Tail returns "ORDER BY xxx LIMIT xxx" SQL query fragment for find and count queries.
```